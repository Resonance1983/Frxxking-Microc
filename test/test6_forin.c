void main() {
    int i;
    for (i = 0; i < 5; i = i + 1)
    {
        print(i);
    }
    println;
    int t;
    for t in range (2,7,2)
    {
        print(t);
    }
}