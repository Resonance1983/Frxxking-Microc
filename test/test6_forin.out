24 19 0 5 25 15 1 13 0 0 1 0 0 12 15 -1 16 43 13 0 0 1 11 22 15 -1 15 0 13 0 0 1 13 0 0 1 11 0 1 1 12 15 -1 13 0 0 1 11 0 5 7 18 18 0 10 23 15 -1 15 1 13 0 1 1 0 2 12 15 -1 16 84 13 0 1 1 11 22 15 -1 15 0 0 2 1 13 0 1 1 11 0 7 7 18 71 15 -2 21 -1